plan.md

```diff
@@ -1,9 +1,282 @@
++ Transparency on business development for all of our team
++ Central location where data can easily be entered as our business development
+efforts expand with [Services](#services) and [Training](#training) offerings
++ 50% of all charitable organisations we have spoken to have told us that their
+current CRM is not fit for purpose, we could easily begin to provide a solution
+to this problem if we desired
+
+> There are many 'small' enhancements we will be making to increase the effectiveness
+of our own workflow. **Each should be evaluated as a potential product.**
+
+
+### Services
+
+Our Services arm is how we help other organisations serve _their_ communities and achieve their social goals.
+
+We *already* do far more than just 'web development' for our services clients
+and this should be formalised into our offering.
+
+#### ATM Workshops  
+Deeply understanding the problem and helping organisations to arrive at a
+solution with actionable metrics is where dwyl adds the most value.
+
+Our 1 to 5 day workshops will be the cornerstone of the Services offering,
+providing clients with:
++ Understanding the "success drivers" i.e. "ATMs"
++ Proposed technical path forward - what technology can the client use to be more effective at reaching their organisational goals
++ Simple prototype
++ Effective [agile collaboration and user testing training](#training)
+
+These workshops will be our largest 'per man hour' contributor to fixed overheads
+from our Services branch by the end of 2017.
+
+#### Product Development
+_Everything_ we build must have a social impact or reusable open source components.
+
+Our [processes](https://github.com/dwyl/contributing) for the delivery of product
+development and [working with services clients](https://github.com/dwyl/ateam-master-reference) are already highly
+developed. These will remain in flux and be [continuously improved]((https://github.com/dwyl/dwylbot)
+as we build our own [products](#product).
+
+#### Focus
++ Delighting our existing customers
++ Writing out the structure for our ATM workshops
++ Building sales through delivering great work and sharing "success stories" (case studies)
++ Creating a simple way for people to share our services
+
+
+#### Metrics to Track in Services
+
++ Did we get a good case study quote from the client/project?
++ Quantitative feedback from clients (particularly if they would work with us again)
++ Sources of new leads (using Google Analytics and conversations, recording
+  this in our CRM)
+  + Number of new leads from recommendations of existing clients
++ Effectiveness of and time spent on individual marketing channels
+(particularly social media)
++ Cost of new client acquisition
++ Net promoter score (NPS)
+
+
+
+### Training
+
+Training is how we share our knowledge with individuals and organisations in our team and community, building them up.
+
+#### Why?
+**Internally**, we want to ensure dwylers continue to learn and grow; facilitating
```

Comment:
dwyl is a [_learning organisation](https://en.wikipedia.org/wiki/Learning_organization)